### PR TITLE
Madhura

### PR DESCRIPTION
**Pull request by mbarve31**
Changed vertex entry point

### CODE REVIEW

**Comment by saurabhjinturkar** [comment]
Please resolve merge conflicts.

File: gradlew.bat [hunk 1]
```diff
@@ -1,84 +1,84 @@
...
+"%JAVA_EXE%" %DEFAULT_JVM_OPTS% %JAVA_OPTS% %GRADLE_OPTS% "-Dorg.gradle.appname=%APP_BASE_NAME%" -classpath "%CLASSPATH%" org.gradle.wrapper.GradleWrapperMain %CMD_LINE_ARGS%
+
+:end
+@rem End local scope for the variables with windows NT shell
+if "%ERRORLEVEL%"=="0" goto mainEnd
+
+:fail
+rem Set variable GRADLE_EXIT_CONSOLE if you need the _script_ return code instead of
+rem the _cmd.exe /c_ return code!
+if  not "" == "%GRADLE_EXIT_CONSOLE%" exit 1
```
Comment (saurabhjinturkar) [inline gradlew.bat]: What is change in this file?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -2,11 +2,11 @@
 ID=node1
 IP=127.0.0.1
 CONNECTED_DEVICE_IDS = node2, node3, node5
-LOCAL_GRAPH=[{"deviceId":"sensor4", "connectedDevices":["sensor2", "sensor3"]},{"deviceId":"sensor5", "connectedDevices":["sensor1", "sensor4"]}, {"deviceId":"sensor3", "connectedDevices":["sensor1", "sensor2"]}]
+LOCAL_GRAPH = [{"deviceId":"sensor10", "connectedDevices":["sensor20", "sensor30"]},{"deviceId":"sensor20", "connectedDevices":["sensor10", "sensor30"]}, {"deviceId":"sensor30", "connectedDevices":["sensor10", "sensor20"]}]
```
Comment (saurabhjinturkar) [inline src/main/resources/application.properties]: Undo these changes. We still want to keep local graph as sensor1,2,3 etc.

**Comment by mbarve31** [comment]
Conflicts Resolved!!

**Comment by saurabhjinturkar** [comment]
Please check other coments.

File: gradlew.bat (hunk 1 above)
Comment (kaustubh-walokar) [inline gradlew.bat]: this is generated by Gradle..manual modifications mustn't be done!
